libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -748,3 +856,12 @@ void RangeFinder::update_pre_arm_check(uint8_t instance)
         state[instance].pre_arm_check = true;
     }
 }
+
+const Vector3f &RangeFinder::get_pos_offset(enum Rotation orientation) const
+{
+    uint8_t i=0;
+    if (find_instance(orientation, i)) {
+        return get_pos_offset(i);
+    }
+    return get_pos_offset(0);
```

Comment:
@OXINARF, the get_pos_offset method is returning a reference to a vector so I think we can't just pass back Vector3f();